### PR TITLE
clear and initialize peers methods

### PR DESCRIPTION
**Pull request by girazoki**

### CODE REVIEW

File: node/src/actors/peers_manager/handlers.rs [hunk 1]
```diff
@@ -24,6 +25,43 @@ impl Handler<AddPeers> for PeersManager {
...
+    type Result = Result<(), failure::Error>;
+
+    fn handle(&mut self, msg: InitializePeers, _: &mut Context<Self>) -> Self::Result {
+        // Clear addresses address
+        log::trace!("Clear all peer addresses from tried bucket");
+        self.peers.clear_tried_bucket();
+        log::trace!("Clear all peer addresses from new bucket");
+        self.peers.clear_new_bucket();
+        log::trace!("Clear all peer addresses from ice bucket");
+        self.peers.clear_tried_bucket();
```
Comment (tmpolaczyk) [inline node/src/actors/peers_manager/handlers.rs:56]: Copy-paste error:

```suggestion
        log::trace!("Clear all peer addresses from ice bucket");
        self.peers.clear_ice_bucket();
```

File: node/src/actors/peers_manager/handlers.rs [hunk 2]
```diff
@@ -24,6 +25,43 @@ impl Handler<AddPeers> for PeersManager {
...
+    type Result = Result<(), failure::Error>;
+
+    fn handle(&mut self, _msg: ClearPeers, _: &mut Context<Self>) -> Self::Result {
+        // Clear addresses address
+        log::trace!("Clear all peer addresses from tried bucket");
+        self.peers.clear_tried_bucket();
+        log::trace!("Clear all peer addresses from new bucket");
+        self.peers.clear_new_bucket();
+        log::trace!("Clear all peer addresses from ice bucket");
+        self.peers.clear_tried_bucket();
```
Comment (tmpolaczyk) [inline node/src/actors/peers_manager/handlers.rs:39]:
```suggestion
        log::trace!("Clear all peer addresses from ice bucket");
        self.peers.clear_ice_bucket();
```

File: src/cli/node/with_node.rs [hunk 3]
```diff
@@ -580,6 +586,21 @@ pub enum Command {
...
     },
+    #[structopt(name = "clearPeers", about = "Clear all peers from the buckets")]
+    ClearPeers {
+        /// Socket address of the Witnet node to query
+        #[structopt(short = "n", long = "node")]
+        node: Option<SocketAddr>,
+    },
+    #[structopt(
+        name = "initializePeers",
+        about = "Clear all peers from the buckets and init  ialize to those in config"
```
Comment (tmpolaczyk) [inline src/cli/node/with_node.rs:597]:
```suggestion
        about = "Clear all peers from the buckets and initialize to those in config"
```

File: node/src/actors/peers_manager/mod.rs [hunk 4]
```diff
@@ -62,6 +62,12 @@ impl PeersManager {
     fn persist_peers(&self, ctx: &mut Context<Self>, storage_peers_period: Duration) {
         // Schedule the discovery_peers with a given period
         ctx.run_later(storage_peers_period, move |act, ctx| {
+            log::error!(
+                "Current tried peers are {:?}",
+                act.peers.get_all_from_tried()
+            );
+            log::error!("Current new peers are {:?}", act.peers.get_all_from_new());
+
```
Comment (tmpolaczyk) [inline node/src/actors/peers_manager/mod.rs:70]: Debug logs?
```suggestion
```

Comment (lrubiorod) [reply node/src/actors/peers_manager/mod.rs:70]: trace better

Comment (lrubiorod) [reply node/src/actors/peers_manager/mod.rs:70]: or remove it :P

**Review by tmpolaczyk — APPROVED** [review]

File: src/cli/node/with_node.rs [hunk 5]
```diff
@@ -580,6 +586,21 @@ pub enum Command {
         #[structopt(name = "peers")]
         peers: Vec<SocketAddr>,
     },
+    #[structopt(name = "clearPeers", about = "Clear all peers from the buckets. These will leave your node without any connections")]
```
Comment (lrubiorod) [inline src/cli/node/with_node.rs:589]: The outbounds and inbounds are not cleared, so technically you still have connections. It only clear the buckets

**Review by lrubiorod — APPROVED** [review]

**Review by mariocao — APPROVED** [review]
Finally, this feature will be available :fireworks: :clap:

**Pull request merged**